web-gateway/app/views/searchItem.scala.html

```diff
@@ -12,6 +12,8 @@
 
     <h2>@message("searchItems")</h2>
 
+	<p>@message("searchInstructions")</p>
```

Comment:
Please be sure to use spaces rather than tabs for indentation.

If you're using Eclipse, here is a guide for configuring it to use spaces: http://stackoverflow.com/a/408449/29470